Doc/library/venv.rst

```diff
@@ -143,7 +143,7 @@ creation according to their needs, the :class:`EnvBuilder` class.
 
 .. class:: EnvBuilder(system_site_packages=False, clear=False, \
                       symlinks=False, upgrade=False, with_pip=False, \
-                      prompt=None, upgrade_deps=False)
+                      prompt=None, upgrade_deps=False, \*, gitignore=False)
```

Comment:
Although it makes sense just to support `git` for now, naming this parameter more generally - say, `add_ignore` - might be better in case we ever decide to support any other ignore files.

Reply:
Don't if this is intended but this is rendered as
<img width="812" alt="Screenshot 2023-08-18 at 19 32 42" src="https://github.com/python/cpython/assets/16805946/c57118f1-c905-4a5b-9aff-17adfdf2aa0f">

i.e. `\*` instead of `*`.

Reply:
Perhaps ``ignore_file``? Though this may harm discoverability---``gitignore`` is very self-describing.

Reply:
I was initially tempted at `ignore_files`, but then I realized it might be **too** general. My concern was let's say Subversion makes a comeback. Would this setting then write out both file or neither file (i.e., all or nothing)? Or would people prefer to have control over each type of ignore file? My thinking landed on the file-specific flag since you can then have more fine-grained control over what's happening.

Not sure if that changes your view at all, @vsajip .

Reply:
I was thinking along the lines of

* `ignore_files=False` -> no ignore files
* `ignore_files=True` -> all ignore files
* `ignore_files=('git', 'svn', ...)` -> just the specified ignore files

Alternatively, don't use `False`/`True` and always require passing either a tuple  or `None`.

Reply:
> Though this may harm discoverability

If you're reading documentation or code to come across the existence of `gitignore` as an option, presumably you'd discover it whatever it was named. Also, we should avoid having to do `gitignore=XXX, svnignore=YYY` etc. in the future if other options become desirable.

Reply:
If we do this then I think the PR should switch to  `--without-ignore_file=...` and use argparse's `"append"` action.